### ISSUE TITLE
nvidia-smi and nvtop shows different results. is libva-nvidia-driver truly working?

### ISSUE THREAD

**Issue by fxzzi** [elFarto/nvidia-vaapi-driver#346]
as shown by `nvtop` here: 
![image](https://github.com/user-attachments/assets/40dd238c-faf3-4993-8976-ff36082a09d9)

there seems to be no decoder usage by any program. Even those which are called N/A (I think the compute N/A is librewolf)

Under `nvidia-smi` it seems to be a similar story:
![image](https://github.com/user-attachments/assets/d1d1371d-c028-4e41-9c11-df5cb20d5cdb)

no compute process is shown here, which is strange.

Under `nvidia-smi pmon` there also seems to be no decoder usage shown:
![image](https://github.com/user-attachments/assets/b9634139-43df-4a20-930c-e4fa0040b4fe)

However, under `nvidia-smi dmon` there is actually decoder usage, does this mean that the driver is working?
![image](https://github.com/user-attachments/assets/b906536b-c21b-4a25-9e8b-10487df68f38)

**Comment by elFarto** [comment]
That's certainly odd. I would say no there's no hardware video decode happening, at least not in librewolf or electron. But as to what is causing the GPU to report that there is decode occurring I have no idea.

**Comment by thesword53** [comment]
Hardware decoding is working:
![image](https://github.com/user-attachments/assets/c7091c35-c444-4b7e-aafa-069ac71ab1ff)
![image](https://github.com/user-attachments/assets/ae41a648-112b-4f33-b578-8a4d479c4b7b)

**Comment by Fxzzi** [comment]
> Hardware decoding is working: ![image](https://private-user-images.githubusercontent.com/66437988/399517340-c7091c35-c444-4b7e-aafa-069ac71ab1ff.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzU2NzM1MzAsIm5iZiI6MTczNTY3MzIzMCwicGF0aCI6Ii82NjQzNzk4OC8zOTk1MTczNDAtYzcwOTFjMzUtYzQ0NC00YjdlLWFhZmEtMDY5YWM3MWFiMWZmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjMxVDE5MjcxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU2NzE4MzIzMGJkMjdkM2QxYWIyYjExYTIxNTdiODExYjgzZGYxNzdhM2EyOWQ1ZjA3ODAyNDU2ZDIxMmQzM2EmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.g3SNVVjFdDzB5I9tOJto1uxGgUUonPtQCDqMPAw7Frw) ![image](https://private-user-images.githubusercontent.com/66437988/399517494-ae41a648-112b-4f33-b578-8a4d479c4b7b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzU2NzM1MzAsIm5iZiI6MTczNTY3MzIzMCwicGF0aCI6Ii82NjQzNzk4OC8zOTk1MTc0OTQtYWU0MWE2NDgtMTEyYi00ZjMzLWI1NzgtOGE0ZDQ3OWM0YjdiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjMxVDE5MjcxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg4ZGJkMTMzNmQ1MGEzYmNlYTFhMzIyZTExNmJlNjMyNmE1ZTY1NWUzNWU5Y2Y2ZDI0NDMzZWZkNTI2NjM5MGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.24n3ZOB_PcNUydBqzYx77aTZPU55rTbRjMFKxtmyb2c)

yeah, you're right. That 9% is the same shown in `nvidia-smi pmon`. And it does go to 0% when the video is paused.

Just a little strange that both `nvidia-smi` and `nvtop` are failing to specify that librewolf is the C process

**Comment by fxzzi** [comment]
I will close this since it does seem like it is making a change to decode usage.

**Comment by fxzzi** [comment]
Yeah, looks like 570 drivers fixed the monitoring issue. Nvidia smi and nvtop both correctly show the compute process now:

![Image](https://github.com/user-attachments/assets/58271a32-3631-46b3-a758-a03e5c30cdda)

![Image](https://github.com/user-attachments/assets/d064bacd-9aa7-4c27-95fe-9dacc1984ce6)
